### DISCUSSION TITLE
Linking field handles in edit forms to their origin?

### DISCUSSION THREAD

**Discussion by ballmannweber** [craftcms/cms#15561] [Ideas]
Wouldn’t it be a great idea to link the field handles in cp’s edit forms directly to their field origin for editing? Especially with large websites it is often very tedious to find out the exact field if you want to modify it. And especially at the beginning of a development, you keep jumping back and forth between the entries and the fields ... and a direct link would save a lot of time.

**Comment by michaelij** [comment]
It isn't native, but the [CP Field Inspect](https://plugins.craftcms.com/cp-field-inspect?craft5) plugin is great for this.
